unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Tests/PlayMode/SettingsSectionTests_PlayMode.cs

```diff
@@ -0,0 +1,65 @@
+using DCL.SettingsPanelHUD.Controls;
+using DCL.SettingsPanelHUD.Sections;
+using DCL.SettingsPanelHUD.Widgets;
+using NSubstitute;
+using NUnit.Framework;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Tests
+{
+
+    public class SettingsSectionTests_PlayMode : TestsBase
```

Comment:
Remember that if the tests don't rely on SceneController, DCLCharacterController, etc. Inheriting from TestsBase is not necessary and we can speed up the test by removing it.

Reply:
Fixed in this new PR (in order to avoid mix things between the part I and II): https://github.com/decentraland/explorer/pull/1784